docs/reference-guides/rancher-security/rancher-v2.6-hardening-guides/rke1-self-assessment-guide-with-cis-v1.6-benchmark.md

```diff
@@ -1,3104 +1,3100 @@
+```
+
+### 1.2.3 Ensure that the --token-auth-file parameter is not set (Automated)
+
+
+**Result:** pass
+
+**Remediation:**
+Follow the documentation and configure alternate mechanisms for authentication. Then,
+edit the API server pod specification file /etc/kubernetes/manifests/kube-apiserver.yaml
+on the master node and remove the --token-auth-file=<filename> parameter.

```

Comment:
```suggestion
on the master node and remove the `--token-auth-file=<filename>` parameter.

```

Converting to inline code since Docusaurus will throw an error because it treats `<filename>` as an HTML tag.

Reply:
resolved